Frameworks/Foundation/NSIndexPath.mm

```diff
@@ -18,6 +18,17 @@
 #import "StubReturn.h"
 #import "Foundation/NSIndexPath.h"
 
+#import <vector>
+#import <numeric>
+
+static NSString* s_archiveKey = @"_NS.IP";
```

Comment:
>"_NS.IP" [](start = 33, length = 8)

nit: is this the standard format we are using?
I recall seeing a different one.